README.md

```diff
@@ -39,15 +39,15 @@ Both architectures handle multiple geo-distributed datacenter outages with a [re
 
 Read the [2PC & Atomizer architecture guide](docs/architecture.md) for a detailed description of the system components and implementation of each architecture.
 
-## Generic Smart Contract Platform ("3PC")
-We built a system that is capable of performing parallel executions of generic smart contracts.
+## Parallel Virtual Machine ("PVM") for Smart Contracts
+We built a system that is capable of performing parallel executions of virtual machines for generic smart contract support.
```

Comment:
How about `We built a system with a generic virtual machine layer that is capable of performing parallel executions of smart contracts`
It may be more correct to say _execution of smart contracts_ than execution of VM's. If we're OK with `execution of VM` then please ignore